### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please don't use git clone or `ldd`.

File: Makefile [hunk 2]
```diff
@@ -26,13 +26,21 @@ SRC = $(LIB_SRC) src/curses.c src/stdout.c src/main.c
 LIB_OBJ = $(LIB_SRC:.c=.o)
```
Comment (xansec) [inline Makefile:26]: This looks acceptable to me.

File: Mayhemfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+image: ghcr.io/capuanob/logtop-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image name.

File: fuzz/fuzz.c [hunk 4]
```diff
@@ -0,0 +1,31 @@
+#include <stdint.h> // uint8_t
```
Comment (xansec) [inline fuzz/fuzz.c]: Can we rename this to indicate what's being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Done!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**